cinema/views.py

```diff
@@ -56,3 +109,24 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+
+class OrderViewSet(viewsets.ModelViewSet):
+    queryset = Order.objects.all()
+    serializer_class = OrderSerializer
+    pagination_class = OrderPagination
+
+    def get_queryset(self):
+        queryset = Order.objects.all()
```

Comment:
I think you have N + 1 problem here because orders also display ticket, movie and cinema_hall data